sample_app/sampleapp_common.c

```diff
@@ -825,7 +825,15 @@ static int app_state_ind (
       (void)pnet_set_provider_state (net, true);
 
       /* Send application ready  */
-      os_event_set (p_appdata->main_events, APP_EVENT_READY_FOR_DATA);
+      if (p_appdata->arguments.verbosity > 0)
```

Comment:
Maybe the old event shall be kept? 
An application that needs additional time to prepare before sending the application ready can then use the existing event to implement this.  This is a comment. I am not sure what is the best solution

Reply:
Ok. 
Done.